apps/ui/lib/api/bot-api/controllers/main.mjs

```diff
@@ -12,7 +17,7 @@ export const main = {
         }),
         (result, res, req) => {
             const chatId = req.chat.id;
-            res.sendMessage(chatId, JSON.stringify(result), { parse_mode: 'HTML' });
+            res.sendMessage(chatId, result, { parse_mode: 'HTML' });
```

Comment:
Is this work after you move out of stringify?

Reply:
yes, otherwise it works with incorrect markup